contents/handbook/strategy/strategy.md

```diff
@@ -5,41 +5,85 @@ showTitle: true
 
 For any company, nothing matters more than product market fit. If we get that right, it’ll be much easier for customer success, marketing or sales teams to succeed.
 
-### Mission
+## Mission
 
 **_“Increase the number of successful products in the world”_**
 
-### Long-term vision (for 2023)
+## Long-term vision (for 2026)
 
-**_“Everyone building a product has a clear path to making it successful without losing control of their data”_**
+**Where do we want to get to?**
 
-This means engineers can install PostHog and within a few hours, qualitative and quantitative data will be captured from their product, audience and any other data sources they have (playing nicely with data warehouses).
+In 2026 we will _go public with $100M ARR._ To achieve this, Posthog will need to be _the standard devtool for building better products._
 
-PostHog will surface this data at the right time to anyone involved in building the product (PMs, Engineers, Support) to make their own well-informed decisions and safely release changes, validate and realize the opportunities identified.
+**How do we get there?**
 
-It will be a “no brainer” for anyone to use PostHog with any product they are building.
+* Build for unserved customers that can’t use cloud offerings because of data control or scale
+* Grow a strong and successful developer community around Posthog
+* Convert our biggest users to paying customers
+* Go broad and develop all the tools our customers need to build better products
+* Partner with others to build plugins and deploy our tools
+* Ride the industry wave of increasing data control that has been triggered by growing privacy regulation
 
-### Target audience (for 2021)
+
+### What should we be working on today?
+
+**The mechanics of success**
+
+Revenue is critical for us to go public _but we won't set short-term revenue targets_ as revenue is an output metric to several other inputs. Instead, we think it is more appropriate to focus on [metrics within our control that contribute more to our revenue in the long-term.](https://app.posthog.com/dashboard/20464)
+
+* **[Quality signups](https://app.posthog.com/insights/ujGv0WqI?events=%5B%5D&actions=%5B%7B%22id%22%3A%2212308%22%2C%22math%22%3A%22dau%22%2C%22name%22%3A%22High%20quality%20sign%20ups%22%2C%22type%22%3A%22actions%22%2C%22order%22%3A0%7D%5D&display=ActionsLineGraph&insight=TRENDS&interval=week&date_from=-90d&new_entity=%5B%5D&properties=%5B%7B%22key%22%3A%22is_organization_first_user%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%5B%22true%22%5D%2C%22operator%22%3A%22exact%22%7D%2C%7B%22key%22%3A%22hubspot_score%22%2C%22type%22%3A%22person%22%2C%22value%22%3A%2270%22%2C%22operator%22%3A%22gt%22%7D%5D&breakdown_type&filter_test_accounts=true#fromDashboard=20464):** Volume of quality organizations signing up 
```

Comment:
Maybe it's worth linking each metric to a specific team that owns it? 

Also maybe just track all these metrics in a single dashboard we link here.

Reply:
I agree - I believe @jamesefhawkins was updating the team charters to link to the relevant goals.

At the start of next year I'd love to facilitate something with every team to renew their charters for the year - and make sure they're really clear on the metrics they're driving and have a dashboard etc. Would love to would with you on this too.